src/api/argo-templates/base/iufBase.template.argo.yaml

```diff
@@ -34,7 +34,26 @@ spec:
       inputs:
        parameters:
          - name: scriptContent
+           description: >
+             The content of the script that will be run
          - name: dryRun
+           description: >
+             A flag indicating whether to run the script or not.
+             True means do not run the script merely output it to a file.
+         - name: script_output_file
+           value: "/tmp/script_output"
```

Comment:
I don't have a good feel for whether this is an issue, so I'll just ask: any concerns about this being overwritten from one use of the template to the next?

Reply:
That file should only live in ephemeral container storage so I think we're good.